src/nwb_datajoint/common/common_sortingview.py

```diff
@@ -0,0 +1,155 @@
+import datajoint as dj
+import numpy as np
+import sortingview as sv
+import spikeinterface as si
+from pathlib import Path
+
+from .common_spikesorting import SpikeSortingRecording, SpikeSorting, Sortings
+
+schema = dj.schema('common_sortingview')
+
+@schema
+class SortingviewWorkspace(dj.Computed):
+    definition = """
+    -> SpikeSortingRecording
+    ---
+    channel = 'franklab2' : varchar(80) # the name of the kachery channel for data sharing
+    sortingview_recording_id: varchar(30)
+    workspace_uri: varchar(1000)
+    """
+    class Sortings(dj.Part):
+        definition = """
+        -> master
```

Comment:
Better to be specific rather than just master

Reply:
changed master to SortingviewWorkspace to be specific